aws-athena/views/default-vw_pin_sale.sql

```diff
@@ -130,7 +130,8 @@ unique_sales AS (
             ) AS sale_filter_deed_type
         FROM {{ source('iasworld', 'sales') }} AS sales
         LEFT JOIN calculated
-            ON sales.instruno = calculated.instruno
+            ON NULLIF(REPLACE(sales.instruno, 'D', ''), '')
```

Comment:
```
calculated AS (
    SELECT
        instruno,
        COUNT(*) AS nopar_calculated
    FROM (
        SELECT DISTINCT
            parid,
            NULLIF(REPLACE(instruno, 'D', ''), '') AS instruno
        FROM {{ source('iasworld', 'sales') }}
        WHERE deactivat IS NULL
            AND cur = 'Y'
    )
    GROUP BY instruno
)
```

Reply:
Yeeep, that'll do it. We'll need to re-ingest sales for modeling.